app/code/Magento/Sitemap/Block/Adminhtml/Edit.php

```diff
@@ -5,28 +5,33 @@
  */
 namespace Magento\Sitemap\Block\Adminhtml;
 
+use Magento\Backend\Block\Widget\Context as WidgetContext;
+use Magento\Backend\Block\Widget\Form\Container;
+use Magento\Framework\Phrase;
+use Magento\Framework\Registry;
+
 /**
  * Sitemap edit form container
  *
  * @author      Magento Core Team <core@magentocommerce.com>
  */
-class Edit extends \Magento\Backend\Block\Widget\Form\Container
+class Edit extends Container
 {
     /**
      * Core registry
      *
-     * @var \Magento\Framework\Registry
+     * @var Registry
      */
     protected $_coreRegistry = null;
 
     /**
-     * @param \Magento\Backend\Block\Widget\Context $context
-     * @param \Magento\Framework\Registry $registry
+     * @param WidgetContext $context
+     * @param Registry $registry
      * @param array $data
      */
     public function __construct(
-        \Magento\Backend\Block\Widget\Context $context,
-        \Magento\Framework\Registry $registry,
+        WidgetContext $context,
+        Registry $registry,
```

Comment:
Why not promote this property?
```suggestion
        protected readonly Registry $_coreRegistry,
```